snap/snapcraft.yaml

```
@@ -0,0 +1,56 @@
+name: chisel
+summary: Chisel is a software tool for carving and cutting Debian packages!
+description: |
+  Chisel can slice a Linux distribution using a release database
+  and construct a new filesystem using the finely defined parts.
```

```suggestion
  Chisel can derive a minimal Ubuntu-like Linux distribution 
  using a release database that defines "slices" of existing packages.
  Slices enable developers to cherry-pick just the files they need 
  from the Ubuntu archives, and combine them to create a new 
  filesystem which can be packaged into an OCI-compliant
  container image or similar.
```

"Finely defined parts" is a bit of a strange read :)

---

Added it in https://github.com/canonical/chisel/pull/92/commits/ca35b7fdd97bab09751cfbdb5280385874ec905e, thanks!

To be honest, I just copy-pasted it from `chisel --help`.